### PR TITLE
Opensource DRagonfly

### PR DESCRIPTION
**Pull request by AdityaStark7**
Hey,
I have added documentation of Dragonfly in Section 5 in  open-source folder of the Opensource Branch.
Kindly Check it!

### CODE REVIEW

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@AdityaStark7 please do these minor changes, rest looks good to me :)

File: Open_Source/Section-5/5.8_DragonFly.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  The goal of Dragonfly is to tackle distribution problems in cloud native scenarios.
+  The project is comprised of three main components:
+   - supernode plays the role of central scheduler and controls all distribution procedure among the peer network 
+   - dfget resides on each peer as an agent to download file pieces 
+   - and “dfdaemon” plays the role of proxy which intercepts image downloading requests from container engine to dfget 
+</p>
+
+# Main Features
+<br>
+<p> This project is an open-source version of the Dragonfly used at Alibaba. You can find it <a href="https://github.com/dragonflyoss/Dragonfly" _target="blank"> here </a> </p>
```
Comment (Kajol-Kumari) [inline Open_Source/Section-5/5.8_DragonFly.md:26]: It should be `target="_blank"` You can read more about it [here](https://www.w3schools.com/tags/att_a_target.asp)

**Comment by AdityaStark7** [comment]
Hey Can you check it now

File: Open_Source/Section-5/5.8_DragonFly.md (hunk 1 above)
Comment (AdityaStark7) [reply Open_Source/Section-5/5.8_DragonFly.md]: Yeah I know. Thanks ! 
Check it Now

File: Open_Source/Section-5/5.8_DragonFly.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+   - and “dfdaemon” plays the role of proxy which intercepts image downloading requests from container engine to dfget 
+</p>
+
+# Main Features
+<br>
+<p> This project is an open-source version of the Dragonfly used at Alibaba. You can find it <a href="https://github.com/dragonflyoss/Dragonfly" target="_blank" rel="noreferrer noopener"> here </a> </p>
+
+<h2> Main Dragonfly Features: </h2>
+<p>
+  - <strong>P2P-based file distribution</strong>: By using the P2P technology for file transmission, it makes the most out of the bandwidth resources of each peer to improve downloading efficiency, and saves a lot of cross-IDC bandwidth, especially the costly cross-board bandwidth.
```
Comment (Kajol-Kumari) [inline Open_Source/Section-5/5.8_DragonFly.md:30]: Screenshot 2021-09-19 at 2 17 11 PM
Bullet isn't appearing for this. Can you please fix it?

**Review by Kajol-Kumari — COMMENTED** [review]
@AdityaStark7 last bit of change required. Rest looks clean and good to go :)

File: Open_Source/Section-5/5.8_DragonFly.md (hunk 2 above)
Comment (AdityaStark7) [reply Open_Source/Section-5/5.8_DragonFly.md]: Done ! Check it Now

**Review by Kajol-Kumari — APPROVED** [review]
Thank you for making the requested changes @AdityaStark7 

LGTM 👍

**Pull request merged**